dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/CpuInfo.java

```diff
@@ -10,20 +10,17 @@
  *******************************************************************************/
 package com.ibm.ws.kernel.service.util;
 
+import java.lang.management.ManagementFactory;
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
-import java.lang.management.ManagementFactory;
-import java.lang.management.OperatingSystemMXBean;
 import java.math.BigDecimal;
 import java.math.RoundingMode;
 
-import javax.management.MBeanServer;
-import javax.management.MalformedObjectNameException;
-import javax.management.ObjectName;
+import com.sun.management.OperatingSystemMXBean;
```

Comment:
This interface does not exist in Java 7.  Do you need to do it the old way with Java 7 and do the lazy initialization logic like @gjdeval did in his PR?

Reply:
It is in my Java SE 7 rt.jar:

com/sun/management/OSMBeanFactory.class
com/sun/management/OperatingSystemMXBean.class
com/sun/management/ThreadMXBean.class

This was hotspot, so I spoke to the runtimes team and they told me it was also in IBM Java 7 which is why I went down this route.

Reply:
My linux IBM Java 7 latest fixpack does not have it.  It is a com.ibm.lang.management interface instead.  I also reviewed Gary's change in PR #7085 and gave him comments.  Likely there should be a merge of your two PRs into something that works on IBM Java 7 and later.  I think what I have put in his review should give you what you are looking for.